README.md

```diff
@@ -1,20 +1,29 @@
 # docs.daml.com
 
-This repo manages the full build of technical documentation into https://docs.daml.com from the following repos:
+This repo manages the [docs.daml.com](https://docs.daml.com) website.
 
-* [Daml docs](https://github.com/digital-asset/daml/tree/main/docs)
-* [Canton docs](https://github.com/DACH-NY/canton/tree/main/docs)
-* [Daml-Finance docs](https://github.com/digital-asset/daml-finance/tree/main/docs)
+In the `docs` folder you will find a folder for each version that is still
+supported by the docs team. Making changes to versions that are not in the docs
+folder is not supported.
 
-:bulb: Docs in this repo build the main TOC with pointers to pages from the above repos.
+Whenever a PR is merged into this repo, the docs website is brought up-to-date
+with the repo. This means that:
 
-The process for updating the docs TOCs has some potential pitfalls. Follow the instructions in this README *carefully* and in order.
+- Every version in the `docs` folder that has changed compared to what is
```

Comment:
This point is unclear. If you make a change to main (i.e. 2.6.0), it sounds like _every_ version changes accordingly. Do you mean that?

Reply:
After careful re-reading, it sounds like when you merge a PR here, if there were also changes to older versions of the docs for some reason, that they all get updated to the versions that are available on the live docs website dropdown? (I could be misreading.) Maybe this would occur if something was also backported? I'm not sure what the concerns would be other than if the backport PRs are not yet intended to go live. It would be good to clarify this situation more.

Reply:
> @da-katmurp 
> main (i.e. 2.6.0)

That is what has changed here: `main` is not just `2.6.0`, `main` has (currently) `2.6.0`, `2.5.2` and `2.5.1`, and may have more in the future.

If you make a PR that changes the `docs/2.5.2` folder and merge that into `main`, CI will update `https://docs.daml.com/2.5.2` to match.

If you've made changes to `docs/2.6.0` and to `docs/2.5.1`, CI will update both `https://docs.daml.com/2.6.0` and `https://docs.daml.com/2.5.1` to match, respectively, the changes in `docs/2.6.0` and the changes in `2.5.1`. In this case, it will not touch the content of `https://docs.daml.com/2.5.2` because `docs/2.5.2` has not changed.

@loiskilberg-da This bullet point has nothing to do with the dropdown. It's concerned with the versions in the `docs` folder.